modules/root/root_api.rb

```diff
@@ -25,4 +28,16 @@ class Proxy::RootApi < Sinatra::Base
       log_halt 400, e
     end
   end
+
+  get "/log_buffer" do
+    begin
+      content_type :json
+      buffer = ::Proxy::LogBuffer::Buffer.instance
+      from_serial = params[:from_serial].to_i
+      records = buffer.to_a(from_serial).collect(&:to_h)
```

Comment:
I don't like the name of this end point, as it refers to the underlying implementation; I'd suggest log, logs. events, etc. Keep in mind we cannot rename this end-point after the api has been made public.